misc/metrics/src/swarm.rs

```diff
@@ -372,3 +383,8 @@ impl<TTransErr> From<&libp2p_swarm::PendingInboundConnectionError<TTransErr>>
         }
     }
 }
+
+fn create_connection_duration_histogram() -> Histogram {
+    // Histogram::new((1..99).map(|i|10. * i as f64))
+    Histogram::new(metrics::histogram::exponential_buckets(1.,2.,10))
```

Comment:
```suggestion
    Histogram::new(metrics::histogram::exponential_buckets(0.001,2.,10))
```

Open Metrics prefers SI base units. See https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#units-and-base-units.